src/chart/drawBars.js

```diff
@@ -362,4 +364,10 @@ export default function drawBars(marks) {
         oldBarGroupsTrans.remove();
         bar_supergroups.remove();
     }
+
+    //Link to the d3.selection from the data
+    bar_supergroups.each(function(d) {
+        d.supergroup = select(this);
+        d.groups = d.supergroup.selectAll('.bar-group');
+    });
```

Comment:
Any reason no `d.bars`?

Reply:
Bars are a bit more confusing since the structure changes when depending on whether they are stacked or grouped. Could add it in later though.

Reply:
Nevermind, explained in the issue.